File: src/main/java/org/example/Java/Act5.java
package org.example.Java;

import java.util.Scanner;

public class Act5 {
    Scanner scanner = new Scanner(System.in);
    public void TablaMultiplicar() {
        System.out.println("Cual es tu numero ? ");
        int number = Integer.parseInt(scanner.nextLine());

        for (int i = 1; i <= 10; i++) {
            System.out.println(number + " x " + i + " = " + (number * i));
        }
    }
}

File: src/main/java/org/example/Kotlin/Act5.kt
package org.example.Kotlin

class Act5 {
    fun TablaMultiplicar() {
        println("Cual es tu numero ? ")
        val number: Int = readLine()!!.toInt()

        for (i in 1..10) {
            println(number.toString() + " x " + i + " = " + (number * i))
        }
    }
}